pkg/job/v1alpha3/kubeflow/kubeflow.go

```diff
@@ -0,0 +1,79 @@
+package kubeflow
+
+import (
+	"fmt"
+
+	commonv1 "github.com/kubeflow/tf-operator/pkg/apis/common/v1"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	logf "sigs.k8s.io/controller-runtime/pkg/runtime/log"
+
+	"github.com/kubeflow/katib/pkg/controller.v1alpha3/consts"
+)
+
+const (
+	PyTorchJobWorkerContainerName = "pytorch"
+
+	TFJobWorkerContainerName = "tensorflow"
```

Comment:
Should we reference this from tf operator repo and PyTorch operator repo?

Reply:
SGTM. I think so.